test_twitter.py

```diff
@@ -34,44 +31,43 @@ def test_upload_photo_correct(local_picture):
     assert returned_name != ''
 
 
-def test_upload_photo_incorrect(local_picture):
-    twitter = Twitter()
+def test_upload_photo_incorrect():
+    twitter = Twitter(upload_photo_mock_request)
     not_picture = "string"
-    with pytest.raises(TypeError):
+    with pytest.raises(TypeError):  # raised by base64
         twitter.upload_photo(name="Test_name", data=not_picture)
 
 
-def test_create_post_correct(picture_id):
-    twitter = Twitter()
-    result = twitter.create_post(status="Test_status", id_of_photo=picture_id)
+def create_post_mock_request(*args, **kwargs):
+    m = mock.MagicMock()
+    m.content = b'''{"id": 243145735212777472}'''
+    return m
+
+
+def test_create_post_correct():
+    twitter = Twitter(create_post_mock_request)
+    result = twitter.create_post(status="Test_status", id_of_photo=1234567)
     assert isinstance(result, int)
 
 
+def get_users_posts_correct_mock_request(*args, **kwargs):
+    m = mock.MagicMock()
+    m.content = b'''[
```

Comment:
Use json.dumps([{'id': 850007368138018817}, {'id': 848930551989915648}]) and move code into the test